src/sentry/utils/committers.py

```diff
@@ -51,7 +51,9 @@ def _get_frame_paths(event):
 def _get_commits(releases):
     return list(
         Commit.objects.filter(
-            releasecommit__in=ReleaseCommit.objects.filter(release__in=releases)
+            releasecommit__in=ReleaseCommit.objects.get_many_from_cache(
+                [r.id for r in releases], key="release"
+            ),
```

Comment:
I'm not totally sure this is better. Previously, this would have done a subquery in postgres, and now it will bring all the release objects into memory, then pass their ids back into postgres to query the commits. It saves on the subquery, but now we're making calls to memcache and also to the database.

If we want to cache this, I'd probably look more to caching all the commits for each release individually, so you'd call `cache.set_many` with something like a map of `{'release_commits:<release_id>': commit_list}`

Reply:
Rewrote this function to change how the caching is being done.